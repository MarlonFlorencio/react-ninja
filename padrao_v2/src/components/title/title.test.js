'use strict'

import React from 'react'
import renderer from 'react-test-renderer'
import Title from './index'

it('Should Title to match snapshot', () => {
  const tree = renderer.create(<Title>Titulo</Title>).toJSON()
    expect(tree).toMatchSnapshot()
})
