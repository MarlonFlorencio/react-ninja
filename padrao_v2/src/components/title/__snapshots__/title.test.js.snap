// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Title to match snapshot 1`] = `
<h1>
  Titulo
</h1>
`;
